web/themes/custom/contribtracker/cypress.config.js

```diff
@@ -1,8 +1,56 @@
 import { defineConfig } from 'cypress';
+import { readFileSync } from 'fs';
+import neatCSV from 'neat-csv';
+import path from 'path';
+import cypressGrep from 'cypress-grep/src/plugin';
+// import cypressSplit from "cypress-split"; // Uncomment if you are using cypress-split
 
 export default defineConfig({
+  projectId: 'contrib-tracker',
+  viewportWidth: 1512,
+  viewportHeight: 1200,
+  watchForFilesChanges: false,
+  chromeWebSecurity: false,
+  defaultCommandTimeout: 60000,
+  execTimeout: 10000,
+  pageLoadTimeout: 500000,
+  requestTimeout: 90000,
+  responseTimeout: 90000,
```

Comment:
These timeouts are really long for CI. Is there a reason for such long timeouts?

Reply:
It is simply a maximum limit; we likely don't want any tests to fail due to exceeding the page load time.